content/en/docs/components/training/overview.md

```diff
@@ -0,0 +1,99 @@
++++
+title = "Overview"
+description = "An overview of Kubeflow Training Operator"
+weight = 5
++++
+
+Kubeflow Training Operator is a Kubernetes-native project for fine-tuning and scalable
+distributed training of machine learning (ML) models created with various ML frameworks such as
+PyTorch, Tensorflow, XGBoost, and others.
+
+Training Operator allows you to use Kubernetes workloads to effectively train your large models
+via Kubernetes Custom Resources APIs or using Training Operator Python SDK.
+
+Training Operator implements centralized Kubernetes controller to orchestrate distributed training jobs.
+
+## Architecture
+
+This diagram shows the major features of Training Operator and supported ML frameworks.
+
+<img src="/docs/components/training/images/training-operator-overview.drawio.png"
+  alt="Training Operator Overview"
+  class="mt-3 mb-3">
+
+## Custom Resources for ML Frameworks
+
+To perform distributed training Training Operator implements the following
+[Custom Resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/):
+
+| ML Framework | Custom Resource |
+| ------------ | --------------- |
+| PyTorch      | PyTorchJob      |
+| Tensorflow   | TFJob           |
+| XGBoost      | XGBoostJob      |
+| MPI          | MPIJob          |
+| PaddlePaddle | PaddleJob       |
+
+You can learn more about each framework in the appropriate guide.
+
+## Getting Started
+
+You can create your first Training Operator job using Python SDK. Define the training function
+that implements end-to-end model training. Training Operator schedules appropriate resources
+to run this training function on every Worker.
+
+Install Training Operator SDK:
+
+```
+pip install kubeflow-training
+```
+
+You can implement your training loop in the train function. Each Worker will execute this function
+on the appropriate Kubernetes Pod. Usually, this function contains logic to download dataset,
+create model, and train the model.
+
+World Size and Rank will be set automatically by Training Operator controller to perform
+[PyTorch DDP](https://pytorch.org/tutorials/intermediate/ddp_tutorial.html).
+
+For example:
+
+```python
+def train_func():
+    import torch
+    import os
+
+    # Create model.
+    class Net(torch.nn.Module):
+        """Create the Pytorch model"""
+        ...
+    model = Net()
+
+    # Download dataset.
+    train_loader = torch.utils.data.DataLoader(...)
+
+    # Attach model to PyTorch distributor.
+    RANK = int(os.environ["RANK"])
+    WORLD_SIZE = int(os.environ["WORLD_SIZE"])
+    torch.distributed.init_process_group(backend="nccl", rank=RANK, world_size=WORLD_SIZE)
```

Comment:
```suggestion
    torch.distributed.init_process_group(backend="nccl", init_method="env://")
```
would be ok.

Reply:
This is great suggestion @kuizhiqing , I didn't know about it!
I guess, the default `init_method="env://"`, so we can even remove it from init_process_group: https://pytorch.org/docs/stable/distributed.html#torch.distributed.init_process_group